test/unit/warp/src/Any.cpp

```diff
@@ -45,18 +46,27 @@ struct AnyMultipleThreadWarpTestKernel
 
         // Some threads quit the kernel to test that the warp operations
         // properly operate on the active threads only
-        if(threadIdxInWarp % 5)
+        if(threadIdxInWarp % 2)
             return;
 
         for(auto idx = 0; idx < warpExtent; idx++)
         {
             ALPAKA_CHECK(*success, alpaka::warp::any(acc, threadIdxInWarp == idx ? 0 : 1) == 1);
-            std::int32_t const expected = idx % 5 ? 0 : 1;
+            std::int32_t const expected = idx % 2 ? 0 : 1;
```

Comment:
Why are those changes necessary?

Reply:
Because SYCL supports also a warpSize of 4 on the CPU and the test fails with 5 instead of 2

Reply:
Ah, ok! Fine then.